Name: Sync more months
Version: 1.31-1
Description: Additional period of days/months to get email adds 2, 3, 6, 9 months and one year back.

--- .orig/usr/palm/applications/com.palm.app.email/app/controllers/accountpreferences-assistant.js
+++ /usr/palm/applications/com.palm.app.email/app/controllers/accountpreferences-assistant.js
@@ -418,6 +418,11 @@
 			{label: $L('7 days'), value: 7},
 			{label: $L('2 weeks'), value: 14},
 			{label: $L('1 month'), value: 30},
+			{label: $L('2 months'), value: 60},
+			{label: $L('3 months'), value: 90},
+			{label: $L('6 months'), value: 180},
+			{label: $L('9 months'), value: 270},
+			{label: $L('1 year'), value: 365},
 			{label: $L('All'), value: 0}
 		]
 	},
